### DISCUSSION TITLE
Animate 64 bit build for desktop？

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#1207] [Q&A]
I use animate CC as the development tool, but when publishing the win runtime package, I always publish 32 bits.

Want to know how to set up a package that can be published as a 64 bit win runtime.

32 or 64 options were not found when publishing embedded runtime applications.

Thank you for your help!

**Comment by ajwfrost** [comment]
See https://github.com/airsdk/Adobe-Runtime-Support/discussions/953#discussioncomment-1395917
I will check with the Animate team on this...

**Reply by Jason0247** [reply]
Thank you！

**Comment by Jason0247** [comment]
@ajwfrost Hello, after testing, animate can publish 64 bit desktop applications, but it is only used after publishing. Using Ctrl + enter to debug still uses 32 bits. I wonder if this problem can be solved from the air SDK.

Thank you for your reply yesterday, thank you!

**Reply by ajwfrost** [reply]
As far as I'm aware it isn't possible to launch the 64-bit version of ADL from within Animate. Weirdly - even though Animate is 64-bit - it fails if I just rename the adl64.exe file to adl.exe so that it tries to launch with this 64-bit version.
